cronos-play/getting-started_unreal/quick-start/erc1155.md

```diff
@@ -30,13 +30,11 @@ Get erc-1155 balance of batch
 * `Success`: whether succeed or not
 * `Output message`: error message, "" if succeed
 
-
-
 ### Erc1155Uri
 
 Get erc-1155 uri
 
-<figure><img src="../../../.gitbook/assets/image (3) (1).png" alt=""><figcaption></figcaption></figure>
+<figure><img src="../../../.gitbook/assets/image (3) (1) (2).png" alt=""><figcaption></figcaption></figure>
```

Comment:
also here and below. Unless there is a reason why we prefer using this format.

Reply:
Renamed.